New package: TurbulenceFlux v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: TurbulenceFlux
- Repository: https://github.com/gabdst/TurbulenceFlux.jl
- Created by: @gabdst
- Version: v0.1.0
- Commit: cb1365cd42e5cfdc443155b9f7b0e9d94013ec09
- Reviewed by: @gabdst
- Reference: https://github.com/gabdst/TurbulenceFlux.jl/commit/cb1365cd42e5cfdc443155b9f7b0e9d94013ec09#commitcomment-148005929
- Description: A Julia package for high resolution flux estimation
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd69e997322d7214c3d0a426b2dd8e33eb3c379c46e5bc674ac44c9504bb1edf0f5e494747a48d42f762ee68c859d0401da3c322e9fc8d00c4f0be57310b1d7da9736ab9f7af150f405ba88b75a7ae55b2783f75abc3fb56b5f77985350293483b793d8eb21b5ce56405f219dc6b2ce9b23207f888ceab3a2d6091338d79c12b406eadf5ec685a21766023ee91e88619f8d6369ef1d1cf6b6abfe738a1d2ba2ff17b377e497d5069f7fc22caea9c84954a5 -->

goerz commented:
Thank you for submitting your package! However, please make sure to add some documentation before registering. [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README

JuliaRegistrator committed:
New package: TurbulenceFlux v0.1.0

UUID: 801b548d-990e-47f8-8ae3-484f64b6e2ab
Repo: https://github.com/gabdst/TurbulenceFlux.jl.git
Tree: 2245dccb314bab3b01486f4a402fe23378a7ba4e

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
